build.gradle

```diff
@@ -25,8 +25,10 @@ repositories {
 }
 
 dependencies {
-    compile 'org.spongepowered:spongeapi:6.0.0-SNAPSHOT'
-    compile 'com.me4502:ModularFramework:1.5'
+    compile 'org.apache.commons:commons-math3:3.6.1'
```

Comment:
Might be worth it to use commons math 4 snapshot, simply for more modern interaction with library(java 5 vs java 8) and more recent updates. Plus it is easier to switch now than later. It seems like it's the main version they are providing information for on their website currently as well. Not all functionality is the exact same though, so it's up to you to decide what you need. 

compile 'org.apache.commons:commons-math4:4.0-SNAPSHOT'